tests/queries/0_stateless/02391_ReplacingCollapsingMergeTree.reference

```diff
@@ -0,0 +1,18 @@
+d1	1	5
```

Comment:
Why do we have 3 times the same d1? 
Why do we have 3 times d2, with different values? etc

Reply:
This "reference" file is the expected output of the {same name}.sql
If we look at the sql file, we will see 3 calls to `select * from test`; that's why. The three output are following without blank/newline in-between. 
So you should have 2 times:
```
d1	1	5
d2	1	1
d3	1	1
d4	1	3
d5	1	1
d6	-1	2
```
and the last insert add overlapping data with new ones, so the output is a little different with
```
d1	1	5
d2	1	3
d3	1	3
d4	1	3
d5	1	1
d6	-1	2
```
new versions of d2 and d3 are in this new batch.

Reply:
oh I somehow missed that there're 3 SELECT calls 👍